### ISSUE TITLE
[Channel] extension that would enhance the navigation on the current site

### ISSUE THREAD

**Issue by jibe-b** [dis-moi/extension#718]
You may propose to the Mozilla community to start a channel that shows the webextensions (addons) that are available and that would improve the navigation on the current webpage.

**Comment by MaartenLMEM** [comment]
I agree on the idea It could be a kind of filter to find addon fitting to my habits. 
But are we the right people to propose that ? Or this proposition would be too much self-serving ?
Also, do you think it s likely to bring real action, regarding the many issues they have ?

**Comment by jibe-b** [comment]
Indeed no, it should not be you maintaining such a channel, it should be the Chromium/Mozilla communities! They have the eye on the webextensions, they promote them so they are the best suited to either maintain such a channel.

**Comment by MaartenLMEM** [comment]
OK ! To be sure... you are talking here about a channel on DisMoi ?

**Comment by jibe-b** [comment]
Yes. Maintained by webextension-catalogs communities (from Mozilla and Chrome)
